packages/runtime-test/src/nodeOps.ts

```diff
@@ -224,8 +224,28 @@ function nextSibling(node: TestNode): TestNode | null {
   return parent.children[i + 1] || null
 }
 
-function querySelector(): any {
-  throw new Error('querySelector not supported in test renderer.')
+function querySelector(selector: string): any {
+  if (selector[0] !== '#') {
+    throw new Error(
+      'querySelector only supports simple IDs in the test renderer.'
```

Comment:
Thoughts on this implementation? It seems like a reasonable restriction for the test renderer.

I don't like that it is dependent on `recordedNodeOps` but it seems like the easiest solution at the moment.